test/unit/org/apache/cassandra/tools/JMXCompatabilityTest.java

```diff
@@ -72,6 +72,7 @@ public class JMXCompatabilityTest extends CQLTester
     @BeforeClass
     public static void setup() throws Exception
     {
+        decorateCQLWithTestNames = false;
```

Comment:
@bereng could you rename this whole test? It is "JMXCompatibilityTest` not `Compatability'.

Reply:
lol will do!